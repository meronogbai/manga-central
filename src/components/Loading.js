import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
import { faSpinner } from '@fortawesome/free-solid-svg-icons';
import styles from '../styles/Loading.module.css';

const Loading = () => (<div className={styles.loading}><FontAwesomeIcon icon={faSpinner} spin size="10x" /></div>);

export default Loading;
