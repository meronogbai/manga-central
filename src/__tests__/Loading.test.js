import { create } from 'react-test-renderer';
import Loading from '../components/Loading';

test('renders correctly', () => {
  const tree = create(<Loading />).toJSON();
  expect(tree).toMatchSnapshot();
});
