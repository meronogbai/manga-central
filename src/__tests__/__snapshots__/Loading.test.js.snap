// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="loading"
>
  <svg
    aria-hidden="true"
    className="svg-inline--fa fa-spinner fa-w-16 fa-spin fa-10x "
    data-icon="spinner"
    data-prefix="fas"
    focusable="false"
    role="img"
    style={Object {}}
    viewBox="0 0 512 512"
    xmlns="http://www.w3.org/2000/svg"
  >
    <path
      d="M304 48c0 26.51-21.49 48-48 48s-48-21.49-48-48 21.49-48 48-48 48 21.49 48 48zm-48 368c-26.51 0-48 21.49-48 48s21.49 48 48 48 48-21.49 48-48-21.49-48-48-48zm208-208c-26.51 0-48 21.49-48 48s21.49 48 48 48 48-21.49 48-48-21.49-48-48-48zM96 256c0-26.51-21.49-48-48-48S0 229.49 0 256s21.49 48 48 48 48-21.49 48-48zm12.922 99.078c-26.51 0-48 21.49-48 48s21.49 48 48 48 48-21.49 48-48c0-26.509-21.491-48-48-48zm294.156 0c-26.51 0-48 21.49-48 48s21.49 48 48 48 48-21.49 48-48c0-26.509-21.49-48-48-48zM108.922 60.922c-26.51 0-48 21.49-48 48s21.49 48 48 48 48-21.49 48-48-21.491-48-48-48z"
      fill="currentColor"
      style={Object {}}
    />
  </svg>
</div>
`;
